packages/react-instantsearch/src/core/InstantSearch.js

```diff
@@ -57,6 +57,9 @@ function validateNextProps(props, nextProps) {
  * @propType {string} appId - The Algolia application id.
  * @propType {string} apiKey - Your Algolia Search-Only API key.
  * @propType {string} indexName - The index in which to search.
+ * @propType {function=identity} configureSearchParameters - function to tweak the parameters sent to Algolia.
```

Comment:
What we want to convey is that by default there's nothing being done. By saying the default value is "identity", any user not knowing what identity functions are (like I was at some point) may be confused.

Should we make sure everyone understands what "identity" means here?